src/test/java/uk/gov/justice/hmpps/casenotes/repository/OffenderCaseNoteRepositoryTest.java

```diff
@@ -43,6 +46,9 @@
 
     @BeforeEach
     public void setUp() {
+        final var jwt = Jwt.withTokenValue("some").subject("anonymous").header("head", "something").build();
+        SecurityContextHolder.getContext().setAuthentication(
```

Comment:
need to actually set with the correct token so user gets injected by the auditor stuff